Makefile

```diff
@@ -80,8 +80,17 @@ TMPFS_MOUNTS := \
 	--tmpfs /tmp:rw,exec,nosuid,size=2097152k
 CERT_MOUNTS := \
 	-v $(CERTS_DIR):/etc/docker/certs.d
-HOME_MOUNTS := \
+
+# The home directory can be mounted as a development convenience.
+# However, on some platforms, where $(HOME) is not set, we default to not mounting anything.
+# Otherwise a mount of `::ro` would be attempted.
+ifdef HOME
```

Comment:
Well, I'm surprised, but it seems like this works: `-v /Users/karl:/Users/karl:ro` (on my mac)